roles/icingadb_redis/templates/icingadb-redis.conf.j2

```diff
@@ -20,6 +20,10 @@ logfile {{ icingadb_redis_logfile }}
 databases {{ icingadb_redis_databases }}
 always-show-logo {{ icingadb_redis_always_show_logo }}
 
+{% if icingadb_redis_password %}
```

Comment:
Can you change the if clause to check if the variable is defined? 
`if icingadb_redis_password is defined ` 
Then we don't need the default variable set to `false`.